src/modules/sign/components/gitSignIn.jsx

```diff
@@ -1,29 +1,46 @@
-import React, {useState} from "react";
+import React, {useContext, useState} from "react";
 import github from "../../../../src/assets/svg/github.svg";
 import activateGithub from "../../../../src/assets/svg/githubActive.svg";
 import gitlab from "../../../../src/assets/svg/gitlab.svg";
 import activateGitlab from "../../../../src/assets/svg/gitlabActive.svg";
+import { StateContext } from "../../../App";
 
 function GitSignIn() {
+  const { dispatch } = useContext(StateContext);
   const [githubActive, githubSetActive] = useState(false);
   const [gitlabActive, gitlabSetActive] = useState(false);
+  const githubActiveStyles = githubActive ? {border: "1px solid black"} : {};
```

Comment:
You can replace `githubActiveStyles` and `gitlabActiveStyles` with CSS `:hover` pseudo class which reduces the number of re-renders of the component and generally cleaner.

Reply:
Okay, Will make the requested changes. Thanks.